tst/pickle.g

```diff
@@ -244,3 +244,58 @@ elif ConjugacyClasses( UnderlyingGroup( tbl ) )
+IS\>0TRANILIS\>3INTG\>13INTG\>12INTG\>11TRANILIS\>3INTG\>11INTG\>11INTG\>11TRANILIS\>3INTG\>13INTG\>13INTG\>13TRANILIS\
+\>0TRANILIS\<67INTG\>11INTG\>12INTG\>13INTG\>14INTG\>15INTG\>16INTG\>17INTG\>18INTG\>19INTG\>1AINTG\>1BINTG\>1CINTG\>1\
+DINTG\>1EINTG\>1FINTG\>210INTG\>211INTG\>212INTG\>213INTG\>214INTG\>215INTG\>216INTG\>217INTG\>218INTG\>219INTG\>21AIN\
+TG\>21BINTG\>21CINTG\>21DINTG\>21EINTG\>21FINTG\>220INTG\>221INTG\>222INTG\>223INTG\>224INTG\>225INTG\>226INTG\>227INT\
+G\>228INTG\>229INTG\>22AINTG\>22BINTG\>22CINTG\>22DINTG\>22EINTG\>22FINTG\>230INTG\>231INTG\>232INTG\>233INTG\>234INTG\
+\>235INTG\>236INTG\>237INTG\>238INTG\>239INTG\>23AINTG\>23BINTG\>23CINTG\>23DINTG\>23EINTG\>23FINTG\>240INTG\>241INTG\
+\>242INTG\>243INTG\>244INTG\>245INTG\>246INTG\>247INTG\>248INTG\>249INTG\>24AINTG\>24BINTG\>24CINTG\>24DINTG\>24EINTG\
+\>24FINTG\>250INTG\>251INTG\>252INTG\>253INTG\>254INTG\>255INTG\>256INTG\>257INTG\>258INTG\>259INTG\>25AINTG\>25BINTG\
+\>25CINTG\>25DINTG\>25EINTG\>25FINTG\>260INTG\>261INTG\>262INTG\>263INTG\>267INTG\>265INTG\>266INTG\>267PPERILIS\>0MLI\
+S\>0PPERILIS\>1INTG\>12MLIS\>1INTG\>13PPERILIS\>1INTG\>33E8MLIS\>1INTG\>33E7PPERILIS\>2INTG\>33E8INTG\>33E9MLIS\>2INTG\
+\>33E8INTG\>13PPERILIS\>3INTG\>19INTG\>1AINTG\>214MLIS\>3INTG\>21EINTG\>228INTG\>232PRMGILIS\>2PERM\>B(1,2,3,4,5)PERM\
+\>5(1,2)INTG\>278FAILPRMGILIS\>2PERM\>7(1,2,3)PERM\>7(4,5,6)FAILFAILPRMGILIS\>0INTG\>11FAILMSTR\>3MaxCHARMCYCL\>4E(4)I\
+NTG\>10GSLPILIS\>2ILIS\>6INTG\>11INTG\>11INTG\>12INTG\>11INTG\>11INTG\>2-1ILIS\>4INTG\>13INTG\>11INTG\>12INTG\>2-1INTG\
+\>12FFEL\>6Z(2)^0FFEL\>6Z(2)^0FFEL\>60*Z(2)FFEL\>6Z(2^3)FFEL\>6Z(2)^0FFEL\>60*Z(2)FFEL\>4Z(3)FFEL\>6Z(3)^0FFEL\>60*Z(3\
+)FFEL\>6Z(3^5)FFEL\>6Z(3)^0FFEL\>60*Z(3)FFEL\>6Z(257)FFEL\>80*Z(257)FFEL\>8Z(257)^0FFEL\>8Z(257,4)FFEL\>80*Z(257)FFEL\
+\>8Z(257)^0FFEL\<12ZmodnZObj(3,65537)FFEL\<12ZmodnZObj(1,65537)FFEL\<12ZmodnZObj(0,65537)FFEL\>AZ(65537,2)FFEL\<12Zmod\
+nZObj(1,65537)FFEL\<12ZmodnZObj(0,65537)PERM\>9(1,2,3,4)GAPL\>4UPOLINTG\>11ILIS\>2ILIS\>3INTG\>11INTG\>11INTG\>11INTG\
+\>10INTG\>11UPOLINTG\>11ILIS\>2ILIS\>8INTG\>11INTG\>10INTG\>10INTG\>11INTG\>10INTG\>10INTG\>10INTG\>11INTG\>2-3INTG\>1\
+1URFUINTG\>11ILIS\>3ILIS\>2INTG\>11INTG\>11ILIS\>2INTG\>12INTG\>11INTG\>10INTG\>11MREC\>2ISTR\>1bMSTR\>3MaxISTR\>1aINT\
+G\>11IREC\>2ISTR\>1cINTG\>13ISTR\>1dISTR\>4TillMF8V\>3FFEL\>4Z(5)FFEL\>60*Z(5)FFEL\>6Z(5)^2IF8V\>3FFEL\>4Z(5)FFEL\>60*\
+Z(5)FFEL\>6Z(5)^2MF8V\>3FFEL\>4Z(7)FFEL\>60*Z(7)FFEL\>6Z(7)^2IF8V\>3FFEL\>4Z(7)FFEL\>60*Z(7)FFEL\>6Z(7)^2MF2V\>2FFEL\>\
+6Z(2)^0FFEL\>60*Z(2)IF2V\>2FFEL\>6Z(2)^0FFEL\>60*Z(2)MF8M\>1MF8V\>3FFEL\>4Z(5)FFEL\>60*Z(5)FFEL\>6Z(5)^2IF8M\>1MF8V\>3\
+FFEL\>4Z(5)FFEL\>60*Z(5)FFEL\>6Z(5)^2MF8M\>1MF8V\>3FFEL\>4Z(7)FFEL\>60*Z(7)FFEL\>6Z(7)^2IF8M\>1MF8V\>3FFEL\>4Z(7)FFEL\
+\>60*Z(7)FFEL\>6Z(7)^2MF2M\>1MF2V\>2FFEL\>6Z(2)^0FFEL\>60*Z(2)IF2M\>1MF2V\>2FFEL\>6Z(2)^0FFEL\>60*Z(2)SREF\>1MREC\>3IS\
+TR\>1xINTG\>11ISTR\>1lSREF\>1ISTR\>1rSREF\<33";
```

Comment:
Any particular reason to store this as a string instead of storing it as a pickle file?

Reply:
No, it saved making an extra file, but that would probably be easier.